### ISSUE TITLE
It's all a bit shit, really!

### ISSUE THREAD

**Issue by guest321** [w4sp-book/w4sp-lab#36]
Bought the book and trying to install the lab.  Errors galore.
I'm clearly not the only person with problems - frustrated at wasting time trying to get a lab to work on an ISO when I'd prefer to spend my time studying, rather than troubleshooting/fixing.
Not having a pop at the helpful people who try to offer good advice in forums for the benefit of their peers.  I do, however, feel let down by the book's writers and publishers.  If this is all so problematic, why don't you just develop your own static ISO with all included, that actually works?  Or provide the pcaps for download?

**Comment by PePeLePuu69** [comment]
Agreed

**Comment by DustyArmstrong** [comment]
I was really looking forward to getting into this, especially with the interactivity of it. But, it seems there are issues with the labs that haven't been resolved. I think it probably would've been best to just host a pre-built ISO with everything required already configured. Most of us I'm sure don't have the skills required to troubleshoot someone else's (extensive) work, if we're reading this sort of book. 

I don't even really know where to begin on resolving it. There were errors in setting up the lab, though it did appear to function, but then attempting to run the "setup" on the localhost just failed utterly. 

I understand that the authors probably have a lot of other things to do than support a book they wrote, but I really do think a pre-built kali would have been the way to go if it was possible.

**Comment by BuhuanJin** [comment]
> 买了这本书并试图安装实验室。错误很多。
> 我显然不是唯一有问题的人 - 当我宁愿花时间学习而不是排除故障/修复时，在试图让实验室参加ISO工作时，我很沮丧。
> 没有流行的有用的人试图在论坛中提供良好的建议，以造福同行。但是，我觉得这本书的作家和出版商感到失望。如果这一切都有问题，你为什么不开发自己的静态ISO ，包括所有，这实际上有效吗？或者提供pcaps下载？

I agree it.

**Comment by gao-xc** [comment]
i feel wasting too much time on trying to solve the problems,,so sad...

**Comment by eiennnohorobi** [comment]
It reminds me that I first install the ubuntu virtual machine on my PC, I spent a lot of time doing this but I can not fetch the lab environment on my PC.

**Comment by anotnow** [comment]
I do wasting too much time on trying the lab also,saddly

**Comment by thedini** [comment]
I reached out to the author on LinkedIn but have not heard back yet. I am trying to build a working VM on an old version of Kali thinking that the newer version of Docker broke something.

**Comment by dbetances3** [comment]
I agree with you. It is really frustrating, we have other classes and we are spending time with something that is it clearly not our fault.
